starmark.dev/src/utils/cssesc.mjs

```
@@ -0,0 +1,89 @@
+// ES module replacement for cssesc
+const cssesc = function(string, options) {
+  options = options || {};
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)